cloud/aws/variables.tf

```diff
@@ -6,6 +6,11 @@ variable "cockroach_port" {
   default = "26257"
 }
 
+# Comma-separated list of directories added as auxiliary stores.
+variable "aux_data_dirs" {
+  default = ""
```

Comment:
because you're assuming that `data` will always be a store. When we get to the point of using multiple real stores per node (soonish probably, I have my eye on a few multi-SSD instances), we'll need to set all the paths, not just the ones other than data.